src/renderer/src/components/Contact/ContactList.tsx

```diff
@@ -0,0 +1,129 @@
+import { Fragment, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TbCheck } from 'react-icons/tb'
+import { IContactState } from '@renderer/@types/store'
+import { SearchInput } from '@renderer/components/SearchInput'
+import { StyleHelper } from '@renderer/helpers/StyleHelper'
+import { useContactsSelector } from '@renderer/hooks/useContactSelector'
+import { cloneDeep } from 'lodash'
+
+import { Separator } from '../Separator'
+
+type TProps = {
+  onContactSelected: (contact: IContactState | null) => void
+  selectFirst: boolean
+  showSelectedIcon: boolean
+  children?: React.ReactNode
+}
+
+export const ContactList = ({ onContactSelected, selectFirst, showSelectedIcon, children }: TProps) => {
+  const { t: contactT } = useTranslation('pages', { keyPrefix: 'contacts' })
```

Comment:
It would be better if this translation was in components instead of pages, since it was changed to be a component